packages/backend/src/services/SpaceService/SpaceService.test.ts

```diff
@@ -0,0 +1,865 @@
+                    user: { organizationRole: OrganizationMemberRole.ADMIN },
+                    space: { isPrivate: true },
+                    access: { groupSpaceRole: SpaceMemberRole.VIEWER },
+                    action: 'manage',
+                    expectedResult: true,
+                    contentType: 'Space',
+                },
+                {
+                    name: 'project admin can manage space even with viewer space role',
+                    user: { projectRole: ProjectMemberRole.ADMIN },
+                    space: { isPrivate: true },
+                    access: { spaceRole: SpaceMemberRole.VIEWER },
+                    action: 'manage',
+                    expectedResult: true,
+                    contentType: 'Space',
+                },
+                {
+                    name: 'project admin can manage space even with viewer group role',
+                    user: { projectRole: ProjectMemberRole.ADMIN },
+                    space: { isPrivate: true },
+                    access: { groupSpaceRole: SpaceMemberRole.VIEWER },
+                    action: 'manage',
+                    expectedResult: true,
+                    contentType: 'Space',
+                },
+                {
+                    name: 'project admin can manage space even with multiple group roles',
+                    user: { projectRole: ProjectMemberRole.ADMIN },
+                    space: { isPrivate: true },
+                    access: {
+                        groupSpaceRoles: [
+                            SpaceMemberRole.VIEWER,
+                            SpaceMemberRole.EDITOR,
+                        ],
+                    },
+                    action: 'manage',
+                    expectedResult: true,
+                    contentType: 'Space',
+                },
+                {
+                    name: 'user with multiple group roles gets highest role (editor over viewer)',
+                    user: { projectRole: ProjectMemberRole.INTERACTIVE_VIEWER },
+                    space: { isPrivate: true },
+                    access: {
+                        groupSpaceRoles: [
+                            SpaceMemberRole.VIEWER,
+                            SpaceMemberRole.EDITOR,
+                        ],
+                    },
+                    action: 'update',
+                    expectedResult: true,
+                    contentType: 'Dashboard',
+                },
+                {
+                    name: 'user with multiple group roles gets highest role (admin over editor)',
+                    user: { projectRole: ProjectMemberRole.INTERACTIVE_VIEWER },
+                    space: { isPrivate: true },
+                    access: {
+                        groupSpaceRoles: [
+                            SpaceMemberRole.EDITOR,
+                            SpaceMemberRole.ADMIN,
+                        ],
+                    },
+                    action: 'update',
```

Comment:
to test this properly we should check action `manage` instead of `update` ,  right ?